commands/tag.js

```diff
@@ -3,9 +3,7 @@ const firebase = require("firebase");
 exports.run = (client, message, args) => {
   let tag = args[0];
 
-  if (tag == null) {
-    return message.channel.send(`You didn't specify a tag!`);
-  }
+  if (!tag) return console.warn("no tag specified using tag command in " + message.guild.name);
```

Comment:
Would you not want to let the user know that they didn't specify a tag? For example if the user executed `+tag` from the users POV it would silently fail?